### ISSUE TITLE
Demo not working anymore

### ISSUE THREAD

**Issue by valerio-bozzolan** [makinacorpus/Leaflet.TextPath#91]
Hi! I've clicked on the demo link in the [README](https://github.com/makinacorpus/Leaflet.TextPath#readme) but it's not working anymore. Are you aware of that?

I was just interested in add an operational demo link from this page:

https://leafletjs.com/plugins.html

Thank you so much for your work on this plugin! :)

**Comment by Bo-Duke** [comment]
Indeed, the link in the Readme pointed to the wrong domain, it's now fixed.
This demo works in https though! :)
